### ISSUE TITLE
Usability/Accessibility: Low contrast

### ISSUE THREAD

**Issue by EnglishRoseBerlin** [swapmyvote/swapmyvote#868]
Some text/elements are currently too low contrast.

Examples below with classes. I'll only give one example of each class if it appears more than once.

Most problems originate from the subdued class colour being too light.

- [ ] Needs to be darker:
  `p class="text-center small subdued"`

  `Matches will be recalculated 30 minutes after they are first shown, to give you a wider range of potential swap partners to choose from.`

- [ ] Needs to be darker:

  `a class="small" href="/contact">Contact Us` (it's the default html styling, but slightly low contrast and blue can be a specific issue)

- [ ] Needs to be darker:

  Published and promoted by Tom de Grunwald on behalf of Swap My Vote, a campaign name of FORWARD DEMOCRACY LIMITED registered at 100 Church Street, Brighton, BN1 1UJ. Registered company in England &amp; Wales no. 11707599.

- [ ] Button needs more contrast - would suggest white text on darker green background or black text on lighter green   background:

  `a class="btn btn-secondary" Offer to swap`

- [ ] Needs to be darker - would check all heading classes:

  `h3 Swap My Vote`

- [ ] Darker, same issue as subdued class above, but mentioning in case you want to style it differently:

  `a class="small subdued logout"  Log out`

- [ ] Ideally darker or bold - mid-blue is less serious on larger elements, but still would advise some caution:

  `span class="name" a href="/user/edit" Name Name a`

**Comment by EnglishRoseBerlin** [comment]
Related...

Here, a mockup. Constituency is using fixed subdued class with one of the BS colours to test it, hence different colour.

I'd emphasise key info a bit more (name, constituency), then add the same info from the chart in text form for screenreaders and people with accessibility issues.

It's not totally elegant. Ideally, that'd be split into two columns below or something. But you get the idea with more emphasis on the constituency and duplicating the info for easy reading and screenreaders. The charts just aren't as user-friendly.

![image](https://github.com/swapmyvote/swapmyvote/assets/104719479/3d9e77ca-2985-4a98-a3d7-c1c7af12fe7a)

**Comment by aspiers** [comment]
@stevebaxter would be amazing if you were able to take this?

**Comment by stevebaxter** [comment]
I've improved the contrast for links and subdued text in #943, it passes Lighthouse now. 

I'm reluctant to change the button colour as it's part of the theme of the site - if we changed the primary buttons to blue we would also want to change links and lots of other things. We're not set up to do that with the way we're using BS, and this is not a change to be making 2 days before the election I feel.

I don't think we should use more space to put the polls in a table. I understand the screen reader aspect, however this is likely to make the experience significantly worse for every other user (this is busy enough as it is). Doing this and adding things like zoom are again not trivial, and not something I would be wanting to do at this stage.
